### ISSUE TITLE
Missing gcc11.patch

### ISSUE THREAD

**Issue by nabobalis** [capitalaslash/archlinux-magics#1]
Am I missing something but compiling this I get:

```
❯ makepkg -si

==> Making package: magics++ 4.8.0-0 (Tue 15 Jun 2021 05:24:54 PM BST)
==> Checking runtime dependencies...
==> Checking buildtime dependencies...
==> Retrieving sources...
  -> Found Magics-4.8.0-Source.tar.gz
==> ERROR: gcc11.patch was not found in the build directory and is not a URL.
```

Should the patch file be in the repo?

**Comment by capitalaslash** [comment]
Sure! Left behind in the commit.
Thanks for signaling.

Solved in d810dff

**Comment by nabobalis** [comment]
Thanks!
